task-launcher/src/tasks/shared/helpers/config.ts

```diff
@@ -91,7 +91,7 @@ export const setSharedConfig = async (firekit: RoarAppkit, gameParams: GameParam
     buttonLayout: buttonLayout || 'default',
     numberOfTrials: numberOfTrials ?? 300,
     task: taskName ?? 'egma-math',
-    stimulusBlocks: stimulusBlocks ?? 3,
+    stimulusBlocks: stimulusBlocks ?? (taskName === 'roar-inference' ? 1 : 3),
```

Comment:
I would just use the url param to control this value instead of changing the default for a specific task.

Reply:
Correct. Is that an issue? We add url params all the time, including taskName etc

Reply:
Alright!